app/ceom/photos/templates/photos/leftmenu.html

```diff
@@ -13,6 +13,13 @@
         <p class="leftmenu-text mx-4 m-0 text-left">Browse</p>
     </a>
  
+</li>
+<li> 
+
```

Comment:
Remove the blank lines in this file
It looks like there were some blank lines from the existing code before you made your changes. You can delete those blank lines as well